book/website/communication/binder/zero-to-binder.md

```diff
@@ -0,0 +1,467 @@
+**Any changes you have made to files inside the Binder will be lost once you close the browser window.**
+```
+
+## 6. Sharing your Work
+
+Binder is all about sharing your work easily and there are two ways to do it:
+
+- Share the **https://mybinder.org/v2/gh/YOU-USERNAME/my-first-binder/HEAD** URL directly
+- Visit **<https://mybinder.org>**, type in the URL of your repo and copy the Markdown or ReStructured Text snippet into your `README.md` file.
+  This snippet will render a badge that people can click, which looks like this: ![Binder](https://mybinder.org/badge_logo.svg)
+
+**TO DO:** :vertical_traffic_light:
+
+1) Add the **Markdown** snippet from **<https://mybinder.org>** to the `README.md` file in your repo
+   - The grey bar displaying a binder badge will unfold to reveal the snippets.
+     Click the clipboard icon next to the box marked with "m" to automatically copy the Markdown snippet.
+2) Click the badge to make sure it works!
+
+## 7. Accessing data in your Binder
+
+Another kind of dependency for projects is **data**.
+There are different ways to make data available in your Binder depending on the size of your data and your preferences for sharing it.
+
+### Small public files
+
+The simplest approach for small, public data files is to add them directly into your GitHub repository.
+They are then directly encapsulated into the environment and versioned along with your code.
+
+This is ideal for files up to **10MB**.
+
+### Medium public files
+
+To access medium files **from a few 10s MB up to a few hundred MB**, you can add a file called `postBuild` to your repo.
+A `postBuild` file is a shell script that is executed as part of the image construction and is only executed once when a new image is built, not every time the Binder is launched.
+
+See [Binder's `postBuild` example](https://mybinder.readthedocs.io/en/latest/using/config_files.html#postbuild-run-code-after-installing-the-environment) for more uses of the `postBuild` script.
+
+```{note}
+New images are only built when Binder sees a new commit, not every time you click the Binder link.
+Therefore, the data is only downloaded once when the Docker image is built, not every time the Binder is launched.
+```
+
+### Large public files
+
+It is not practical to place large files in your GitHub repo or include them directly in the image that Binder builds.
+The best option for large files is to use a library specific to the data format to stream the data as you're using it or to download it on demand as part of your code.
+
+For security reasons, the outgoing traffic of your Binder is restricted to HTTP or GitHub connections only. You will not be able to use FTP sites to fetch data on mybinder.org.
+
+### Private files
+
+There is no way to access files which are not public from mybinder.org.
+You should consider all information in your Binder as public, meaning that:
+
+- there should be no passwords, tokens, keys etc in your GitHub repo;
```

Comment:
This etc. is causing the "no bad latin" check to fail

Reply:
Thank you - this is still a draft PR and I haven't tweaked all the text yet 😊